ci/ray_ci/BUILD.bazel

```diff
@@ -201,3 +201,23 @@ py_test(
         ci_require("pytest"),
     ],
 )
+
+py_test(
+    name = "test_upgrade_version",
+    size = "small",
+    srcs = ["test_upgrade_version.py"],
```

Comment:
the test here need to depend on the `ugprade_version.py` file?

I guess it is included in the `ray_ci_lib` now.

maybe exclude the `upgrade_version.py` in `ray_ci` lib, or put it in another dir.

Reply:
done!